schemas/api/errors.schema.json

```
@@ -41,6 +43,14 @@
                 "AccessDenied",
                 "CreationFailed"
             ]
+        },
+        "BridgingError": {
+            "type": "string",
+            "description": "Contains constants representing the errors that can be encountered by Desktop Agents when exchanging bridging messages.",
+            "enum": [
+                "ResponseToBridgeTimedOut",
+                "AgentDisconnected"
```

```suggestion
                "AgentDisconnected",
                "NotConnectedToBridge"
```

Added another error in spec, for when you try to target something against other agents and are not connected to bridge